src/views/ADempiere/Window/windowUtils.js

```diff
@@ -86,22 +80,10 @@ export function generateTabs({
       tabParentIndex
     }
   })
-
-  // generate tabs childs
-  const tabsListChild = tabsList.filter(tabItem => {
```

Comment:
this is used to render the child tabs in
https://github.com/adempiere/adempiere-vue/blob/window-redefinition/src/views/ADempiere/Window/StandardWindow.vue#L31